src/View/Admin/brand/add.html.twig

```diff
@@ -0,0 +1,16 @@
+
+{% extends 'Admin/layoutAdmin.html.twig' %}
+
+{% block content %}
+    {% if error > 0%}
```

Comment:
essaie un if error not empty plutot 
https://twig.symfony.com/doc/2.x/tests/empty.html